ctapipe/resources/base_config.yaml

```diff
@@ -93,9 +93,9 @@ ImageProcessor:
 # The ShowerProcessor performs the DL1 to DL2a (reconstructed shower geometry)
 # transition. It is run only if the parameter DataWriter.write_showers=True.
 ShowerProcessor:
-  # choose between HillasReconstructor and HillasIntersection (two
+  # choose between HillasReconstructor and HillasIntersection or both (two
   # implementations of the standard stereo line-intersection method)
-  reconstructor_type: HillasReconstructor
+  reconstructor_types: [HillasReconstructor, HillasIntersection]
```

Comment:
In YAML this is maybe better written as:
```yaml
reconstructor_types:
    - HillasReconstructor
    - HillasIntersection
```

Reply:
also not sure if the default config should really include both (remember this config is what a user will get when running ctapipe-quickstart), so perhaps comment-out the HillasIntersection line just to show that it can be enabled, but is not enabled by default